scripts/lint-style.lean

```diff
@@ -36,13 +37,14 @@ def explicitImports : Array Lean.Name := #[`Batteries, `Std]
 def eraseExplicitImports (names : Array Lean.Name) : Array Lean.Name :=
   explicitImports.foldl Array.erase names
 
-/-- Check that `Mathlib.Init` is transitively imported in all of Mathlib.
+/-- Check that `Mathlib.Init` is transitively imported in all of Mathlib:
+if not, return the number of modules which did not import `Mathlib.Init`.
 
 Every file imported in `Mathlib.Init` should in turn import the `Header` linter
 (except for the header linter itself, of course).
 Return `true` iff there was an error.
```

Comment:
```suggestion
Return `0` iff there are no errors.
```
or some reformulation of what you wrote in the first paragraph.

Reply:
Actually, looking at the implementation, this function returns
* the number of modules importing the header linter, unless this number is zero;
* the number of modules not importing the mathlib, otherwise.

Should this function return an `IO (Nat \times Nat)` instead, one for each count?

Reply:
That makes it hard to add up the numbers: how about just returning the sum? (Changed to that, and fixed the doc-string.)